vyper/cli/vyper_serve.py

```diff
@@ -66,7 +66,7 @@ def do_GET(self):
             self.send_response(200)
             self.send_cors_all()
             self.end_headers()
-            self.wfile.write('Vyper Compiler. Version: {} \n'.format(vyper.__version__).encode())
+            self.wfile.write('Vyper Compiler. Version: {vyper.__version__).encode()} \n')
```

Comment:
Was a paren missing here?
```suggestion
            self.wfile.write('Vyper Compiler. Version: {(vyper.__version__).encode()} \n')
```